Cargo.toml

```diff
@@ -11,7 +11,7 @@ edition = "2018"
 categories = ["cryptography::cryptocurrencies", "development-tools::testing"]
 
 [dependencies]
-bitcoincore-rpc = { version = "0.18.0" }
+bitcoincore-rpc = { version = ">= 0.17.0, <= 0.19" }
```

Comment:
Note, some nuance around the version number 0.19.0:
- that version does not exist (but cargo is ok with non-existent value in the range)
- that version has been created locally so we have some confidence that it will exist and will not break the build
- we don't use some random high number because we don't want to break `bitcoind` by an unrelated later release of `bitcoincore-rpc`
- we have to do a point release bumping the top of the range every time we have an upcoming `bitcoincore-rpc` release (implies everytime `rust-bitcoin` is releaseing)